op-node/node/node.go

```diff
@@ -78,6 +80,9 @@ func (n *OpNode) init(ctx context.Context, cfg *Config, snapshotLog log.Logger)
 	if err := n.initL1(ctx, cfg); err != nil {
 		return err
 	}
+	if err := n.initRuntimeConfig(ctx, cfg); err != nil {
+		return err
```

Comment:
Maybe an error log here to suggest that the user should check the supplied L1 RPC url + connection? That seems like the most likely reason for `initRuntimeConfig` to fail.

Reply:
I think it's better if we separate the L1 identity check, and then we can check chain ID and rollup gensis block presence etc.
I've opened a linear issue to track it: ENG-3134